dts/bindings/i2s/nxp,mcux-i2s.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2021, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP mcux SAI-I2S controller
+
+compatible: "nxp,mcux-i2s"
+
+include: [i2s-controller.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    dma-names:
+      required: true
+
+    nxp,tx-dma-channel:
+      type: int
+      required: true
+      description: tx dma channel number
+
+    nxp,rx-dma-channel:
+      type: int
+      required: true
+      description: rx dma channel number
+
+    nxp,tx-sync-mode:
+      type: boolean
+      required: false
+      description: tx sync mode
+
+    nxp,rx-sync-mode:
+      type: boolean
+      required: false
+      description: rx sync mode
+
+    pre-div:
+      type: int
+      required: true
+      description: pre divider
+
+    podf:
+      type: int
+      required: true
+      description: post-divider fraction
+
+    pll-clocks:
+      type: phandle-array
+      required: true
+      description: pll settings
+      specifier-space: pll-clock
+
+    pll-clock-names:
+      required: false
+      type: string-array
+      description: Provided names of pll-clock specifiers
+
+    pinmuxs:
+      type: phandle-array
+      required: true
+      description: iomux settings
```

Comment:
```suggestion
    pinmuxes:
      type: phandle-array
      required: true
      specifier-space: pinmux
      description: iomux settings
```

Reply:
Looking at this with fresh eyes now that pinctrl has been merged I want to ask @gmarull if this looks like the sort of thing that should be in a pinctrl property now.